### DISCUSSION TITLE
How do I remove the default colors (`emerald` etc.)

### DISCUSSION THREAD

**Discussion by hampusborgos** [chakra-ui/panda#2599] [Q&A]
I am using Panda, and enjoy it a lot. However I would like to restrict the color palette to the tokens available in the design system of the product.

Currently when I use `'@shadow-panda/preset'` you get a bunch of colors (inherited from Panda) that I would like to disable, like `emerald`, `rose`, `pink` etc.

Is there a way to remove these colors so they do not show up as autocomplete with `strictTokens: true`?

**Comment by kiruushaaa** [comment]
`{ theme: { tokens: { colors: { ... } } } }` will overwrite tailwinds' color palette

**Reply by astahmer** [reply]
that works, but note that [this will also overwrite other `tokens`](https://panda-css.com/docs/overview/faq#how-can-i-completely-override-the-default-tokens) (see also the [Minimal setup page](https://panda-css.com/docs/guides/minimal-setup#removing-default-tokens))

another way to do that is by [tweaking the resolved config using hooks](https://panda-css.com/docs/concepts/hooks#tweak-the-config) so you can selectively remove some stuff
